lib/beaker/host/windows/pkg.rb

```diff
@@ -25,15 +22,6 @@ def install_package(name, cmdline_args = '')
       cygwin = "setup-x86.exe"
     end
 
-    if not check_for_command(cygwin)
```

Comment:
Is this removed code landing in a new method... or does it already exist elsewhere already?